src/pages/about.tsx

```diff
@@ -1,6 +1,280 @@
 import React from 'react';
 import { ComingSoon } from '@components';
 
+// Migrate info from about.
+// eslint-disable-next-line
+const TEAM = [
```

Comment:
Isn't this inside the `team-profile` folder ?

Reply:
Oops. Removed